projects/or_scs/src/Mirza/OrgRegistry/Types.hs

```diff
@@ -157,10 +160,22 @@ instance ToParamSchema AuthUser
 
 
 data PartialNewOrg = PartialNewOrg
-  { partialNewOrgName      :: Text
-  , partialNewOrgUrl       :: Network.URI.URI
-  } deriving (Generic, Eq, Show)
-$(deriveJSON defaultOptions ''PartialNewOrg)
+  { partialNewOrgName :: Text
+  , partialNewOrgUrl  :: Network.URI.URI
+  }
+  deriving (Generic, Eq, Show)
+
+instance FromJSON PartialNewOrg where
+  parseJSON = withObject "PartialNewOrg" $ \o -> PartialNewOrg
+    <$> o .: "org_name"
```

Comment:
I'd prefer to drop the prefix, unless this is a convention we already have?

The URL, i.e. the resource, which I'm POSTing gives enough context to know that the `name` field is that of the organisation without needing a prefix.